src/components/common/TextInput.vue

```diff
@@ -16,59 +19,93 @@
 </script>
 
 <template>
-	<div id="container" :class="{ focused: inFocus }" >
-		<label :for="props.name">{{ props.name }}</label>
-		<input
-			type="text"
-			:name="props.name"
-			:placeholder="props.name"
-			@focusin="toggleFocus(true)"
-			@focusout="toggleFocus(false)"
-		/>
-		<button @click="$emit('on-click')" >
-			<font-awesome-icon icon="fa-solid fa-arrow-right" />
-		</button>
+	<div id="textinput-container">
+		<div id="textinput-header" :class="{ focused: inFocus }">
+			<h3>Create</h3>
+		</div>
+		<div id="text-container" :class="{ focused: inFocus }">
+			<label :for="props.name">{{ props.name }}</label>
+			<input
```

Comment:
Text input height should be the same a the containers height so that the full create area if clickable for text input

Reply:
Agreed, but punting to later because this is a frustrating fix
- getting ref in components API outside of `setup()` seems impossible
- fixing height constraint is proving difficult

will address later